### ISSUE TITLE
GUI lagging with "Gelände" and Rendering V2 active

### ISSUE THREAD

**Issue by Isolator70** [osmandapp/OsmAnd#17032]
### Description
Graphical User Interface shows strong lagging with "Gelände" and Rendering engine V2 active. This can be seen in map scrolling (offline vector tiles) but surprisingly also in some sub menus of the app as well and also while dragging the side bar over the stationary map.

### Steps to reproduce
Switch on Rendering Engine V2, activate "Gelände" in map settings and slowly scroll map, drag out left side bar - see attached screenshot movie.

### Actual result
The GUI is stuttering for ~ 0.5s with 2-5s cycle time.

### Expected result
The GUI runs smoothly

### Your Environment
OsmAnd Version: 4.4.7
Android/iOS version: Android 9.0
Device model: Huawei P10 (VTR-L29)

https://user-images.githubusercontent.com/35434137/233936364-e0abab78-1d30-4f18-9fa6-093e4673ec12.mp4

**Comment by scaidermern** [comment]
"Gelände" is "Terrain" aka hillshade.

**Comment by Isolator70** [comment]
"Gelände" translated to "Terrain"

**Comment by Isolator70** [comment]
> Did you change anything in development settings

No, I've just activated it. But if you can tell me what settings I could check I can deliver feedback.

**Comment by vshcherb** [comment]
There are settings how to render Hillshade / Slopes from new 3D maps or as old Terrain sqlite maps but probably it's unrelated to your case cause 3D maps are even slower
